### ISSUE TITLE
Second page

### ISSUE THREAD

**Issue by scinfu** [t413/SinglePaged#48]
Hi,
how to have 2 or more pages in the same site?

**Comment by DavidBurela** [comment]
agreed. It is a great "single page" template. but would still be great to be able to add sub pages like /about
